helm/values.yaml

```diff
@@ -0,0 +1,92 @@
+bootstrap:
+  image:
+    #must be postgres or mysql
+    repository: postgres
+    pullPolicy: IfNotPresent
+    tag: 16.1-bullseye
+
+image:
+  repository:
+  pullPolicy: IfNotPresent
+  tag:
+
+gateway:
+  requestPort: 8080
+  appPort: 8090
+  adminPort: 8091
+  loggers:
+    level: INFO
+  log:
+    level: INFO
+  tls:
+    enabled: false
+    keystorePath: ""
+    keystorePassword: ""
+  routingRules:
+    enabled: false
+    configPath: ""
```

Comment:
How about adding a configmap for this and provide default routing rules as routing rules is almost mandatory to use gateway.

Reply:
configmap added